config/dev.env.js

```diff
@@ -4,5 +4,6 @@ const prodEnv = require('./prod.env')
 
 module.exports = merge(prodEnv, {
   NODE_ENV: '"development"',
-  HPO_SOLR_ENDPOINT: '"/solr/hpo-pl/select"'
+  HPO_SOLR_ENDPOINT: '"/solr/hpo-pl/select"',
+  GENOMICS_API_ENDPOINT: '"http://localhost:4000/bodyHPOSearch/saveSearchResults"'
```

Comment:
You probably need to change `config/prod.env.js` as well.

Reply:
checked in PROD config